src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/Types/boolean.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if attribute.value %}
+    {{ 'sylius.ui.yes_label'|trans }}
+{% else %}
+    {{ 'sylius.ui.no_label'|trans }}
```

Comment:
Why `yes_label` and `no_label`? Cannot it just be `yes` and `no`?

Reply:
I used existed message in translation.